docs/src/examples/ls.md

```diff
@@ -0,0 +1,85 @@
+# A regularized least-square problem
+
+In this tutorial, we will show how to model and solve the nonconvex nonsmooth least-square problem
+```math
+  \min_{x \in \mathbb{R}^n} \frac{1}{2} \|Ax - b\|_2^2 + \lambda \|x\|_0.
+```
+
+## Modelling the problem
+We first formulate the objective function as the sum of a smooth function $f$ and a nonsmooth regularizer $h$:
+```math
+  \frac{1}{2} \|Ax - b\|_2^2 + \lambda \|x\|_0 = f(x) + h(x),
+```
+where 
+```math
+\begin{align*}
+f(x) &:= \frac{1}{2} \|Ax - b\|_2^2,\\
+h(x) &:= \lambda\|x\|_0.
+\end{align*}
+```
+
+To model $f$, we are going to use [LLSModels.jl](https://github.com/JuliaSmoothOptimizers/LLSModels.jl).
+For the nonsmooth regularizer, we observe that $h$ is actually readily available in [ProximalOperators.jl](https://github.com/JuliaFirstOrder/ProximalOperators.jl), you can refer to [this section](@ref regularizers) for a list of readily available regularizers.
```

Comment:
There's really no need to say this. We *only support* regularizers from ProximalOperators.jl.